CHANGES.md

```diff
@@ -60,6 +60,7 @@ Core Grammars:
 - fix(yaml) - Fixed wrong escaping behavior in single quoted strings [guuido]
 - enh(nim) - Add `concept` and `defer` to list of Nim keywords [Jake Leahy]
 - fix(cpp) - Exclude keywords from highlighting as function calls [Eisenwave]
+- enh(cpp) add C++26 keywords, `#embed`, and compiler-builtin types [Eisenwave]
```

Comment:
You also need to add your footnote on line ~90...

Reply:
It's already there.

Reply:
Ah, I see.  I thiknk you need to move it up to the `Version 11.11.2` section though now that time has moved on.